databricks/koalas/plot.py

```diff
@@ -992,6 +992,18 @@ def line(self, x=None, y=None, **kwargs):
         See Also
         --------
         matplotlib.pyplot.plot : Plot y versus x as lines and/or markers.
+
+        Examples
+        --------
+        Basic plot.
+
+        .. plot::
+            :context: close-figs
+
+            >>> import matplotlib.pyplot as plt
+            >>> plt.close('all')
```

Comment:
Do we still need this?
Seems like others don't have this, otherwise we need to add for the others?

Reply:
@ueshin at above discussions, maybe there is an issue that `:context: close-figs` doesn't work properly. so now it wouldn't work if we remove the line `plt.close('all')` to explicitly close all plots.  i'm going to figure out how can we solve this problem.

Reply:
@itholic 
I think `:context: close-figs` is just for Sphinx, and does nothing when pytest runs tests.

Reply:
So when running tests, all the plots are drawn onto the same `ax` and get messed up.

Reply:
@harupy Thanks for comment!! It really makes sense. Then i think maybe we need to skip these all examples on pytest, and added them separate unittest (actually most of them already tested in unittest suite though)

Reply:
I added `plt.savefig` to see what's happening and ran `./dev/pytest -k plot.py --doctest-modules databricks`.
```
# doctest of KoalasSeriesPlotMethods.line

>>> import matplotlib.pyplot as plt
>>> plt.savefig('foo.png')
>>> s = ks.Series([1, 3, 2])
>>> ax = s.plot.line()
``` 

And I got the figure below.

![foo](https://user-images.githubusercontent.com/17039389/65960447-78397b00-e48f-11e9-8cd2-6267945aecc1.png)

Reply:
@harupy Thanks for researching! 😄  now i think it is clear that `close-fig` doesn't work in pytest. (maybe this syntax only for sphinx). isn't it?

Reply:
Adding the code below solves this issue.

```python
@pytest.fixture(autouse=True)
def close_figs():
    yield
    plt.close("all")
```

This is actually what `seaborn` does.(https://github.com/mwaskom/seaborn/blob/master/seaborn/conftest.py#L6)

Reply:
@harupy Genius, It's works! Thanks for researching really so much !! 😺

Reply:
but I'm not sure if this is right because `plt.close("all")` is called on tests which are not related to plots.

Reply:
ah? sorry i can't get it perfectly 😢 maybe could you let me know where they are using for tests not related to plots?

Reply:
`plt.close` is called on test functions where we actually don't need to call `plt.close`.

Reply:
got it :)
but maybe i think they are only called when we explicitly write the `:context: close-figs` in doctest, ain't they?

Reply:
no